### ISSUE TITLE
No Module find error

### ISSUE THREAD

**Issue by SajjadIqbal-git** [MuhammadMoinFaisal/YOLOv8-object-tracking-blurring-counting#2]
"""Traceback (most recent call last):
  File "predict.py", line 13, in <module>
    from ultralytics.yolo.engine.predictor import BasePredictor
ModuleNotFoundError: No module named 'ultralytics.yolo'"""
Got this error while running the predict.py file. Tried different ways but didn't resolve the issue. What could be the reason?
What changes you guys can suggest? I implemented this on my friend's laptop, and it was working fine there but on my laptop, I got this error. Need urgent assistance. @MuhammadMoinFaisal Thank you.

**Comment by mobina-zibandehpoor** [comment]
Yes, this is the same problem for me, what must I do? @MuhammadMoinFaisal

**Comment by SajjadIqbal-git** [comment]
> Yes, this is the same problem for me, what must I do? @MuhammadMoinFaisal
@Mobina-Zibandepour : try one of these will solve your problem. 
"pip install -e ."or  "pip install setup.py"
